.github/workflows/release.yaml

```diff
@@ -3,19 +3,19 @@ on:
     types: [created]
 name: Handle Release
 jobs:
-  linux:
+  build-bin:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Go
-      uses: actions/setup-go@v2
-      with:
-        go-version: 1.16.5
-    - name: Make Directory with kgctl Binaries to Be Released
-      run: make release
-    - name: Publish Release
-      uses: skx/github-action-publish-binaries@master
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        args: 'bin/release/kgctl-*'
+      - uses: actions/checkout@v2
```

Comment:
I think these white-space changes don't need to be in this PR. Please revert this file and then we can merge the PR :rocket:

Reply:
fixed, many thanks for you